testsuite/ui/views/devel/settings/stripe.py

```diff
@@ -1,7 +1,7 @@
 """Settings Devel portal View containing credit card details for Stripe payment gateway"""
+# pylint: disable=invalid-name
```

Comment:
???

Reply:
What is more preferable? To use module-level pylint ignore or use the same one on 3 different places in a module?

Reply:
Definitely 3 in the exact spots they are needed